### DISCUSSION TITLE
Is Reflection implemented in COSMOS?

### DISCUSSION THREAD

**Discussion by SirBrahms** [CosmosOS/Cosmos#2183] [Q&A]
Can I use Reflection in my project?

**Comment by quajak** [comment]
No

**Reply by SirBrahms** [reply]
Alright, thanks!

**Comment by jwoff78** [comment]
Okay, I agree with with @quajak says, but what's reflection for? If it's for modifying cosmos you should look into plugs or fork cosmos and edit it yourself.

**Reply by SirBrahms** [reply]
I wanted to use it for dynamically calling functions at runtime, but I'm sure you can use it for other neat types of stuff.

Also thank you, I'll look into that!
